pkg/cmd/kafka/list/list.go

```diff
@@ -71,8 +71,8 @@ func NewListCommand(f *factory.Factory) *cobra.Command {
 		Example: opts.localizer.MustLocalize("kafka.list.cmd.example"),
 		Args:    cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, flagutil.ValidOutputFormats...) {
-				return flagutil.InvalidValueError("output", opts.outputFormat, flagutil.ValidOutputFormats...)
+			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, flagutil.ValidListOutputFormats...) {
```

Comment:
We also have:  flagutil.ValidateOutputForTable . Might be worth to use this across or even remove that method if not needed.

Reply:
yes, I did use it here: pkg/cmd/kafka/topic/list/list.go in the latest commit as well. Will look for other suitable places as well